assets/src/application/ui/forms/espressoForm/renderers/FieldRenderer.tsx

```diff
@@ -11,6 +11,11 @@ const FieldRenderer: React.FC<FieldRendererProps> = (props) => {
 
 	const formItemLayout = useFormItemLayout();
 
+	// no layout stuff needed for hidden field
+	if (props.fieldType === 'hidden') {
+		<MappedField {...rest} />;
```

Comment:
is it missing a return here or I don't fully understand what is the intention, because below we have another:
`<MappedField {...rest} />` inside `FormItem`

Reply:
Already pushed a fix